BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Help/SkontoHelpViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SkontoHelpViewModel.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+
+struct SkontoHelpViewModel {
+    let items: [SkontoHelpItem]
+
+    init() {
+        let item1 = SkontoHelpItem(
+            icon: GiniImages.skontoHelpItem1.image,
+            title: NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.help.content.item1",
+                                                            comment: "Use your camera to capture invoices with cash discounts")
+        )
+
+        let item2 = SkontoHelpItem(
+            icon: GiniImages.skontoHelpItem2.image,
+            title: NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.help.content.item2",
+                                                            comment: "Immediately identify discounted amounts and due dates")
+        )
+
+        let item3 = SkontoHelpItem(
+            icon: GiniImages.skontoHelpItem3.image,
+            title: NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.help.content.item3",
+                                                            comment: "Approve or adjust payment details")
+        )
+
+        let item4 = SkontoHelpItem(
+            icon: GiniImages.skontoHelpItem4.image,
+            title: NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.help.content.item4",
+                                                            comment: "Save money by paying promptly")
+        )
```

Comment:
I'd prefer to keep them as short names with order, because its used only inside one code block and main idea is to init all of them in correct order. Let me know please if its ok

Reply:
maybe we don't need at all this.
Why not do this:
```
let skontoHelpItemsData = [
    (icon: GiniImages.skontoHelpItem1.image, 
     titleKey: "ginibank.skonto.help.content.item1", 
     comment: "Use your camera to capture invoices with cash discounts"),
     
    (icon: GiniImages.skontoHelpItem2.image, 
     titleKey: "ginibank.skonto.help.content.item2", 
     comment: "Immediately identify discounted amounts and due dates"),
     
    (icon: GiniImages.skontoHelpItem3.image, 
     titleKey: "ginibank.skonto.help.content.item3", 
     comment: "Approve or adjust payment details"),
     
    (icon: GiniImages.skontoHelpItem4.image, 
     titleKey: "ginibank.skonto.help.content.item4", 
     comment: "Save money by paying promptly")
]

items = skontoHelpItemsData.map { data in
    SkontoHelpItem(
        icon: data.icon,
        title: NSLocalizedStringPreferredGiniBankFormat(data.titleKey, comment: data.comment)
    )
}
```

Reply:
Here we're creating an array of untyped objects and then we map them into typed ones (whats require iteration through all of them)
Not sure its optimised solution from algo side